docs/example/example/processes/template_py_process.py

```diff
@@ -0,0 +1,285 @@
+"""Describe what tasks this Process accomplishes."""
+# Order of imported packages should be:
+# Standard libraries
+from pathlib import Path
+from shutil import copy2
+
+# Third party libraries
+from plumbum import TEE
+
+# Resolwe
+from resolwe.process import (
+    Process,
+    SchedulingClass,
+    Persistence,
+    Cmd,
+    StringField,
+    TextField,
+    BooleanField,
+    IntegerField,
+    FloatField,
+    DateField,
+    DateTimeField,
+    UrlField,
+    SecretField,
+    FileField,
+    FileHtmlField,
+    DirField,
+    JsonField,
+    ListField,
+    DataField,
+    GroupField,
+)
+
+# In-house libraries (i.e. resolwe_bio)
+# Local libraries
+
+
+class YourProcessName(Process):
+    """Process description in one line ends with a period.
+
+    Continue with a more elaborate description of what the process
+    does, short layout of the algorithm and what output(s) to expect.
+
+    Note that this is not a working process, but rather just shows
+    bits and pieces that can be used in your real process.
+    """
+
+    slug = "process-slug-name"
+    name = "Human readable name of the process"
+    process_type = "data:text_processing:"
+    version = "1.0.0"
+    # A category of the process, can be used in i.e. frontend app.
+    category = "Read Processing"
+    # Dynamically generated name of the resulting datum object. Lookup
+    # happens from Input class fields.
+    data_name = 'Data object ({{ string_field.file|default("?") }})'
+    scheduling_class = SchedulingClass.BATCH  # or .INTERACTIVE for short jobs
+    # or RAW or TMP, see documentation at
+    # https://resolwe.readthedocs.io/en/latest/proc.html#persistence
+    persistence = Persistence.CACHED
+    # Use below entity to make the created object also an entity of
+    # type sample. See
+    # https://resolwe.readthedocs.io/en/latest/proc.html#entity for
+    # more details.
+    # Fields descriptor_schema, input and always_create are optional.
+    entity = {
+        "type": "sample",  # name of your entity
+        # Use a descriptor schema slug if it differs from "type".
+        "descriptor_schema": "descriptor-schema-of-entity",
+        "input": "name of input that has the entity",
+        # If you want to create entity in all cases
+        "always_create": True,
+    }
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {
+                # Refer to a valid Docker image. If using process
```

Comment:
This is not resolwe-bio specific and depends on settings: https://github.com/genialis/resolwe/blob/640bda7f97c2afca1d2223dc038352e8364f9331/tests/settings.py#L147

Reply:
Comment made agnostic.